app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,123 @@
+        D[{{site.base_gateway}}]
+        E[{{site.mesh_product_name}}]
+        F[{{site.konnect_short_name}} Analytics]
+        G[API Products]
+    end
+
+    subgraph "External Integrations"
+        H[GitHub]
+        I[PagerDuty]
+    end
+
+    D -->A[{{site.konnect_short_name}} Service Catalog]
+    E -->A
+    F -->A
+    G -->A
+
+    H -->A
+    I -->A
+{% endmermaid %}
+
+> Figure 1: This diagram shows how you can pull services from built-in integrations (like {{site.base_gateway}}, {{site.mesh_product_name}}, {{site.konnect_short_name}} Analytics, and API Products) and external services (like from GitHub and PagerDuty) into the Service Catalog. This consolidates all your organization's services and service health metrics into a single view.
+
+Specifically, Service Catalog addresses the following problems:
+
+* **Service discovery:** Gain visibility into all your services, including unrecognized or undiscovered APIs that live in your organization’s fragmented ecosystems. 
+* **Service ownership:** Keep track of your organization's service ownership by mapping teams and services. 
+* **Service health:** The Service Catalog consolidates key service health metrics into a single view, so you no longer need to check multiple tools (like GitHub and PagerDuty) for recent activity and status. It also helps you quickly identify and reuse existing services, reducing duplication and effort.
+* **Service governance and policy enforcement:** Enables governance in how services are created and maintained across your company to ensure you adhere to security, compliance, and engineering best practices.
+
+{% mermaid %}
+graph TD
+    subgraph "{{site.konnect_short_name}} Service Catalog"
+        SC[Service Catalog]
+        SG[{{site.base_gateway}} Service]
+        API[API Product]
+    end
+
+    subgraph "External APIs"
+        GitHub[GitHub]
+        PagerDuty[PagerDuty]
+        Data[External API Data]
+    end
+
+    GitHub -->|Pulls Service| SC
+    PagerDuty -->|Pulls Service| SC
+    SC -->|Maps to| SG
+    SC -->|Maps to| API
+    SC -->|Collects Data from| Data
+    Data -->|Reports to| SC
+{% endmermaid %}
+
+> Figure 2: This diagram shows how you can map services from one source, like GitHub, to another, such as a {{site.base_gateway}} service, in Service Catalog. Service Catalog also shows you how your services interact with external APIs.
+
+## Service Catalog terminology
+
+| Term | Definition |
+| ---- | ---------- |
+| Service Catalog | The canonical system of record for your organization's services in {{site.konnect_short_name}}. This includes all the interactions between them and with external APIs. |
+| [Service](/gateway/latest/key-concepts/services/) | A service is the top-level entity in the Service Catalog. It typically represents an external upstream API or microservice that's owned by a singular team in your organization. For example, a data transformation microservice, a billing API, and so on. |
```

Comment:
Would suggest changing the link here, since a Service Catalog Service is not the same thing as a Gateway Service!